### PR TITLE
created fix for stlloader discussed at #11272, but messed up other branch as well as #11584...

### PR DESCRIPTION
**Pull request by cnspaha**

### CODE REVIEW

File: examples/js/loaders/STLLoader.js [hunk 1]
```diff
@@ -245,22 +279,13 @@ THREE.STLLoader.prototype = {
 		if ( typeof buf !== "string" ) {
 
 			var array_buffer = new Uint8Array( buf );
+			var strArray = [];
+			for ( var i = 0; i < buf.byteLength; i ++ ) {
 
-			if ( window.TextDecoder !== undefined ) {
```
Comment (mrdoob) [inline examples/js/loaders/STLLoader.js]: Hmm... Why did you removed the `TextDecode` code?

Comment (cnspaha) [inline examples/js/loaders/STLLoader.js]: sorry.. worked with a little previous version.. reverted this and commited only the fix. sorry 'bout that

**Comment by mrdoob** [comment]
Nice!

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**